docs/protocol/specifications/data.md

```diff
@@ -499,14 +505,12 @@ Root hash of the [merkle binary hash tree](#merkle-binary-tree) formed by the *t
 
 Root hash of the [merkle patricia tree](#merkle-patricia-tree) formed by unspent outputs with an **asset version 1** after applying the block. Allows bootstrapping nodes from recent blocks and an archived copy of the corresponding merkle patricia tree without processing all historical transactions.
 
-The tree contains [non-retired](#retired-asset) unspent outputs (one or more per [asset ID](#asset-id)):
+The tree contains [non-retired](#retired-asset) unspent outputs (one or more per [asset ID](#asset-id)) where both key and value are the same value — the [Unspent ID](#unspent-id) of the unspent output.
```

Comment:
Do we need to redefine this as a set anyway? We can't change it later without breaking all the existing commitments right?